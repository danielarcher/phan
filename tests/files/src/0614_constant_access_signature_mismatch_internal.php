<?php

class MyPDO extends PDO {
    private const FETCH_OBJ = 1;
}
