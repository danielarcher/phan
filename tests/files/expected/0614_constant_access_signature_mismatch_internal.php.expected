%s:4 PhanConstantAccessSignatureMismatchInternal Access level to private \MyPDO::FETCH_OBJ must be compatible with internal public \PDO::FETCH_OBJ